janet-bistro.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Is this Janet Bistro page supposed to be here?
Were you creating this page thinking you were on another branch?